Tests/SonarScanner.MSBuild.PreProcessor.Test/TargetsInstallerTests.cs

```diff
@@ -25,403 +25,377 @@
-        public void InstallTargetsFile_Overwrite()
-        {
-            // In case the dummy targets file somehow does not get deleted (e.g. when debugging), make sure its content valid XML
-            var sourceTargetsContent1 = @"<Project ToolsVersion=""4.0"" xmlns=""http://schemas.microsoft.com/developer/msbuild/2003"" />";
-            var sourceTargetsContent2 = @"<Project ToolsVersion=""12.0"" xmlns=""http://schemas.microsoft.com/developer/msbuild/2003"" />";
+        var msBuildPathSettings = new MsBuildPathSettings();
 
-            CreateDummySourceTargetsFile(sourceTargetsContent1);
+        InstallTargetsFileAndAssert(sourceTargetsContent1, expectCopy: true);
+        msBuildPathSettings.GetImportBeforePaths().Should().HaveCount(7, "Expecting six destination directories");
 
-            var msBuildPathSettings = new MsBuildPathSettings();
+        var path = Path.Combine(msBuildPathSettings.GetImportBeforePaths().First(), FileConstants.ImportBeforeTargetsName);
+        File.Delete(path);
 
-            InstallTargetsFileAndAssert(sourceTargetsContent1, expectCopy: true);
-            msBuildPathSettings.GetImportBeforePaths().Should().HaveCount(7, "Expecting six destination directories");
+        CreateDummySourceTargetsFile(sourceTargetsContent2);
+        InstallTargetsFileAndAssert(sourceTargetsContent2, expectCopy: true);
+        InstallTargetsFileAndAssert(sourceTargetsContent2, expectCopy: true);
+    }
 
-            var path = Path.Combine(msBuildPathSettings.GetImportBeforePaths().First(), FileConstants.ImportBeforeTargetsName);
-            File.Delete(path);
+    [TestMethod]
+    public void InstallLoaderTargets_GlobalTargets_Exist()
+    {
+        // Arrange
+        var targetsInstaller = new TargetsInstaller(logger, msBuildPathSettingsMock, fileWrapperMock, directoryWrapperMock);
+        msBuildPathSettingsMock.GetGlobalTargetsPaths().Returns(["global"]);
+        fileWrapperMock.Exists("global\\SonarQube.Integration.ImportBefore.targets").Returns(true);
 
-            CreateDummySourceTargetsFile(sourceTargetsContent2);
-            InstallTargetsFileAndAssert(sourceTargetsContent2, expectCopy: true);
-            InstallTargetsFileAndAssert(sourceTargetsContent2, expectCopy: true);
+        // Act
+        using (new AssertIgnoreScope())
+        {
+            targetsInstaller.InstallLoaderTargets("c:\\project");
         }
 
-        [TestMethod]
-        public void InstallLoaderTargets_GlobalTargets_Exist()
-        {
-            // Arrange
-            var targetsInstaller = new TargetsInstaller(this.logger, this.msBuildPathSettingsMock.Object,
-                this.fileWrapperMock.Object, this.directoryWrapperMock.Object);
+        // Assert
+        logger.Warnings.Should().Contain(m => m.StartsWith("This version of the SonarScanner for MSBuild automatically deploys"));
```

Comment:
CaYC, as you've modified the line... This can be done in the whole file
```suggestion
        logger.Warnings.Should().Contain(x => x.StartsWith("This version of the SonarScanner for MSBuild automatically deploys"));
```